hoomd/hpmc/HPMCCounters.h

```diff
@@ -388,6 +343,8 @@ DEVICE inline hpmc_implicit_counters_t operator+(const hpmc_implicit_counters_t&
     {
     hpmc_implicit_counters_t result;
     result.insert_count = a.insert_count + b.insert_count;
+    result.insert_accept_count = a.insert_accept_count + b.insert_accept_count;
+    result.insert_accept_count_sq = a.insert_accept_count_sq + b.insert_accept_count_sq;
```

Comment:
Squared counts do not add and subtract. Why do you need to store squared counts in addition to counts?

Reply:
This was an attempt at calculating the variance of the energy estimator. It will go away in the next PR.

Reply:
Why not just remove it from this PR then?

Reply:
@tommy-waltmann in the interests of keeping the PRs smaller we are going to merge some changes only to undo them later.  I didn't realize that this was one of those changes. I'm fine with adding it now - the beta releases are unstable.